client/java/src/main/java/com/linecorp/centraldogma/client/CentralDogma.java

```diff
@@ -517,8 +566,10 @@
      * <p>Note that {@link Function} by default is executed by a blocking task executor so that you can
      * safely call a blocking operation.
      */
-    <T, U> Watcher<U> fileWatcher(String projectName, String repositoryName,
-                                  Query<T> query, Function<? super T, ? extends U> function);
+    default <T, U> Watcher<U> fileWatcher(String projectName, String repositoryName,
+                                  Query<T> query, Function<? super T, ? extends U> function) {
```

Comment:
It seems like the indentation is wrong. Could you check your settings? 😉 
https://armeria.dev/community/developer-guide#setting-up-your-ide